test-data/unit/check-attr.test

```diff
@@ -0,0 +1,160 @@
+[case testUntypedAttrS]
+import attr
+@attr.s
+class UnTyped:
+    normal = attr.ib()
+    _private = attr.ib()
+    def_arg = attr.ib(18)
+    _def_kwarg = attr.ib(validator=None, default=18)
+
+    CLASS_VAR = 18
+reveal_type(UnTyped)  # E: Revealed type is 'def (normal: Any, private: Any, def_arg: Any =, def_kwarg: Any =) -> __main__.UnTyped'
+UnTyped(1, 2)
+UnTyped('1', '2', '3')
+UnTyped('1', '2', '3', 4)
+UnTyped('1', '2', def_kwarg='5')
+UnTyped('1', '2', def_kwarg='5')
+UnTyped(normal=1, private=2, def_arg=3, def_kwarg=4)
+UnTyped()  # E: Too few arguments for "UnTyped"
+UnTyped(1, 2) == UnTyped(2, 3)
+UnTyped(1, 2) >= UnTyped(2, 3)
+[builtins fixtures/attr_builtins.pyi]
+[add-module fixtures/attr.pyi]
+
+[case testUntypedNoUntypedAttrS]
+# flags: --disallow-untyped-defs
+import attr
+@attr.s
+class UnTyped:
+    normal = attr.ib()  # E: Need type annotation for variable
+    _private = attr.ib()  # E: Need type annotation for variable
+    def_arg = attr.ib(18)  # E: Need type annotation for variable
+    _def_kwarg = attr.ib(validator=None, default=18)   # E: Need type annotation for variable
+
+    CLASS_VAR = 18
+[builtins fixtures/attr_builtins.pyi]
+[add-module fixtures/attr.pyi]
+
+[case testTypedAttrS]
```

Comment:
This and some other tests are too large. Try splitting it in half (like the previous test).